stanford/roles/cluster/defaults/main.yml

```diff
@@ -335,3 +345,5 @@ CLUSTER_ELBS:
             cidr_ip: '0.0.0.0/0'
       subnets:
         - "{{ COMMON_DEPLOYMENT }}-subnet-public-{{ CLUSTER_NUMBER }}"
+
+INSTALL_INSTANCES: false
```

Comment:
I understand why we're doing this in a separate PR from your other one.
One downside here, however, is that it's not clear to me that this will fully suit our needs in the other PR.

Specifically, I don't think we want a boolean value here. We presumably would want more than two possible values here; `ec2` vs `rds` vs `emr` as opposed to simply `ec2` or not.

Perhaps this should be something like:
```yaml
CLUSTER_SERVICE_TYPE: 'ec2'
# alternate values are
# CLUSTER_SERVICE_TYPE: 'rds'
# CLUSTER_SERVICE_TYPE: 'emr'
# etc.
```

Reply:
In this spirit, maybe we should include, at the least, some placeholders for the anticipated services.
This could be files like:

## rds.yml
```yaml
- debug: msg="TODO: Flesh this out"
```

## emr.yml
```yaml
- debug: msg="TODO: Flesh this out"
```

Reply:
These will come with my next pr.